draft-ietf-httpbis-priority.md

```diff
@@ -642,6 +642,23 @@ signal the intended priority to the client by including the Priority field in
 a PUSH_PROMISE or HEADERS frame.
 
 
+## Intermediaries with Multiple Backend Connections
+
+An intermediary serving an HTTP connection might split requests over multiple
+backend connections. When it applies prioritization rules strictly, low priority
+requests cannot make progress while requests with priorities are inflight. This
```

Comment:
```suggestion
requests cannot make progress while requests with higher priorities are inflight. This
```
@LPardue Just making sure that the omission of "higher" is intentional, and flows more naturally to a native (for me, either works).

Reply:
it wasn't intentional sorry, good catch!